.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "ccp/submodules/kubic-project-automation"]
+	path = ccp/submodules/kubic-project-automation
+	url = https://github.com/kubic-project/automation.git
```

Comment:
tracking master can lead to problems in the long term, can we tie it to a specific branch? I don't remember if you can point to a tag or a specific commit id though